Tests/DataPipeline/Util/DataPipelinesLoggerTests.swift

```diff
@@ -0,0 +1,66 @@
+@testable import CioDataPipelines
+@testable import CioInternalCommon
+import Foundation
+import SharedTests
+import XCTest
+
+class DataPipelinesLoggerTests: UnitTest {
+    private let loggerMock = LoggerMock()
+    var logger: DataPipelinesLogger!
+
+    override func setUp() {
+        super.setUp()
+
+        logger = DataPipelinesLoggerImpl(logger: loggerMock)
+    }
+
+    func test_logStoringDevicePushToken_logsExpectedMessage() {
+        let token = "token"
+        let userId = "user-id"
+
+        logger.logStoringDevicePushToken(token: token, userId: userId)
+
+        XCTAssertEqual(loggerMock.debugReceivedInvocations.count, 1)
+        XCTAssertEqual(loggerMock.debugReceivedInvocations.first?.tag, "Push")
+        XCTAssertEqual(
+            loggerMock.debugReceivedInvocations.first?.message,
+            "Storing device token: token for user profile: user-id"
+        )
+    }
+
+    func test_logStoringBlankPushToken_logsExpectedMessage() {
```

Comment:
we should also probably test that `logRegisteringPushToken` was not called for this?